x-pack/solutions/security/packages/kbn-securitysolution-io-ts-list-types/src/common/timestamp/index.ts

```diff
@@ -6,7 +6,12 @@
  */
 
 import * as t from 'io-ts';
-import { IsoDateString } from '@kbn/securitysolution-io-ts-types';
+import { IsoDateString, PositiveInteger } from '@kbn/securitysolution-io-ts-types';
 
 export const timestamp = IsoDateString;
 export const timestampOrUndefined = t.union([IsoDateString, t.undefined]);
+
+/**
+ * timestamp field type as it can be returned form ES: string, number or undefined
+ */
+export const timestampFromEsResponse = t.union([IsoDateString, PositiveInteger, t.undefined]);
```

Comment:
Since we are working on this area already, what are your thoughts on including negative integers? I don't think we explicitly guard against negative integers anywhere so I feel this type could be misleading to only include `PositiveInteger`

Reply:
@dhurley14, good point. I changed to `number`